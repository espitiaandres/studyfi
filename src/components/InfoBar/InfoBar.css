.infoBar {
  align-items: center;
  background: #1ED760;
  border-radius: 4px 4px 0 0;
  display: flex;
  justify-content: space-between;
  height: 60px;
  width: 100%;
}
  
.leftInnerContainer {
  align-items: center;
  color: white;
  display: flex;
  flex: 0.5;
  margin-left: 5%;
  width: 100%;
}
  
.rightInnerContainer {
  display: flex;
  flex: 0.5;
  justify-content: flex-end;
  margin-right: 5%;
  width: 100%;
}

.onlineIcon {
  margin-right: 5%;
}

.seasonStyling {
  background: #F37D0F;
}

.seasonStylingAlt {
  background: #FFE01B;
}
