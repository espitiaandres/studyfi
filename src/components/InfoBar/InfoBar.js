//
//  InfoBar.js
//  react-spotify-player
//
//  Created by Andres Espitia.
//  Copyright Â© 2020 Andres Espitia. All rights reserved.
//

import React from 'react';
import closeIcon from '../../icons/closeIcon.png';
import onlineIcon from '../../icons/onlineIcon.png';
import './InfoBar.css';

const InfoBar = ({ room, season }) => {
    let seasonStyling = season ? "seasonStyling" : "";
    // let seasonStylingAlt = season ? "seasonStylingAlt" : "";

    return (
        <div className={`infoBar ${seasonStyling}`}>
            <div className="leftInnerContainer">
                <img className="onlineIcon" src={onlineIcon} alt="online"/>
                <h3>{room}</h3>
            </div>
            <div className="rightInnerContainer">
                <a href="https://www.andres-espitia.com/studyfi/#">
                    <img src={closeIcon} alt="close" />
                </a>
            </div>
        </div>
    )
}

export default InfoBar;
