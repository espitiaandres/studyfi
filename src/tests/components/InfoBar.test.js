//
//  InfoBar.test.js
//  react-spotify-player
//
//  Created by Andres Espitia.
//  Copyright Â© 2020 Andres Espitia. All rights reserved.
//

import React from 'react';
import { shallow } from 'enzyme';
import InfoBar from '../../components/InfoBar/InfoBar';

test('should render InfoBar component correctly', () => {
    const wrapper = shallow(<InfoBar />);
    expect(wrapper).toMatchSnapshot();
});
