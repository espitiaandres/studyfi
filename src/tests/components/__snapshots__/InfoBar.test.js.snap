// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render InfoBar component correctly 1`] = `ShallowWrapper {}`;
